src/stirling/source_connectors/socket_tracer/grpc_trace_bpf_test.cc

```diff
@@ -140,16 +140,13 @@ TEST_P(GRPCTraceTest, CaptureRPCTraceRecord) {
   server_.LaunchServer(params.go_version, params.use_https);
 
   // Deploy uprobes on the newly launched server.
-  RefreshContext(/* blocking_deploy_uprobes */ true);
-
-  StartTransferDataThread();
+  ASSERT_OK(source_.RefreshContextAndDeployUProbes());
```

Comment:
Need to rm this LOC and retry the test, i.e. because refresh & deploy is inherent to `Start` now.